Nautono/RTOS_framework_v1/src/sail_i2c.c

```diff
@@ -135,15 +144,27 @@ static void configure_i2c(void) {
 	config_i2c_master.buffer_timeout = 65535;
 	
 	// Select SERCOM port
-	config_i2c_master.pinmux_pad0 = SERCOM2_PAD0_DEFAULT;
-	config_i2c_master.pinmux_pad1 = SERCOM2_PAD1_DEFAULT;
+	#ifdef PCB
```

Comment:
We should update this build macro to illustrate that this is for the 1st PCB revision

Reply:
I suggest adding a new header file where all of our macros that modify the build behaviour such as PCB rev or Sensor tests. Maybe call it:
```
build.h
```
or 
```
config.h
```

Reply:
I'm think this could a whole task, where you and I clean up the code base and move some things around. Thoughts?